aim/web/ui_v2/src/pages/Params/Params.tsx

```diff
@@ -35,7 +38,17 @@ const Params = (
           <Grid container className={classes.fullHeight} spacing={1}>
             <Grid item xs>
               <Paper className={classes.paper}>
-                <HighPlot index={0} />
+                <HighPlot index={0} curveInterpolation={curveInterpolation} />
+              </Paper>
+            </Grid>
+            <Grid item>
+              <Paper className={classes.paper}>
+                <Controls
+                  curveInterpolationChangeHandler={
+                    curveInterpolationChangeHandler
```

Comment:
Can you please change curveInterpolationChangeHandler with onCurveInterpolationChange ?. With front team we chose this type of naming for component handlers